src/components/binding/binding-loader.js

```diff
@@ -31,16 +32,23 @@ genotet.BindingLoader.prototype.LOCUS_MARGIN_RATIO = .1;
  * @param {string} fileName Binding file name.
  * @param {string} bedName Bed name of the bed binding track.
  * @param {string} chr ID of the chromosome.
+ * @param {number} numberOfTracks Number of tracks.
  * @param {number=} opt_track Track # into which the data is loaded.
  * @override
  */
 genotet.BindingLoader.prototype.load = function(fileName, bedName, chr,
-                                                opt_track) {
+                                                numberOfTracks, opt_track) {
   var trackIndex = opt_track ? opt_track : this.data.tracks.length;
   this.data.chr = chr;
   this.loadFullTrack(trackIndex, fileName, chr);
   this.loadBed(bedName, chr, this.data.detailXMin, this.data.detailXMax);
   this.loadExons_(chr);
+
+  this.data.defaultNumberOfTracks = numberOfTracks;
```

Comment:
Because when we create only one new track at first, this.data.overviewRangeChanged is true and this.getBindingRanges_() works. Then when we add more tracks manually, this.data.overviewRangeChanged is false when no range changed. However, when we create more than one new tracks at first, this.data.overviewRangeChanged is false after loading and sent to renderer. So that this.getBindingRanges_() doesn't work and the view doesn't render correctly. Therefore, I set up this variable to ensure this.getBindingRanges_() works no matter how many tracks are loaded at first. Actually, this variable can be just set as a boolean simply, which works the same.